test/radio/group-spec.js

```diff
@@ -334,4 +334,14 @@ describe('Radio.Group', () => {
         });
     });
 
+    describe('Children over datasource', () => {
+        it('children is rendered when provided', () => {
```

Comment:
stick with the `it('should ...')` pattern for test names

Reply:
ok, I'll change it to ```suggestion
        it('should render children over datasource', () => {
```